qa/integration-steps/src/main/java/org/eclipse/kapua/qa/integration/steps/DockerSteps.java

```diff
@@ -0,0 +1,423 @@
+        String containerId = mbContainerCreation.id();
+
+        docker.startContainer(containerId);
+        docker.connectToNetwork(containerId, networkId);
+        containerMap.put(bcData.getName(), containerId);
+        logger.info("Message Broker {} container started: {}", bcData.getName(), containerId);
+    }
+
+    @Then("^Stop container with name \"(.*)\"$")
+    public void stopContainer(String name) throws DockerException, InterruptedException {
+        logger.info("Stopping container {}...", name);
+        String containerId = containerMap.get(name);
+        docker.stopContainer(containerId, 3);
+        logger.info("Container {} stopped.", name);
+    }
+
+    @Then("^Remove container with name \"(.*)\"$")
+    public void removeContainer(String name) throws DockerException, InterruptedException {
+        logger.info("Removing container {}...", name);
+        String containerId = containerMap.get(name);
+        docker.removeContainer(containerId);
+        logger.info("Container {} removed.", name);
+    }
+
+    /**
+     * Creation of docker container configuration for broker.
+     *
+     * @param brokerAddr
+     * @param brokerIp
+     * @param clusterName
+     * @param controlMessageForwarding
+     * @param mqttPort                 mqtt port on docker
+     * @param mqttHostPort             mqtt port on docker host
+     * @param mqttsPort                mqtts port on docker
+     * @param mqttsHostPort            mqtts port on docker host
+     * @param webPort                  web port on docker
+     * @param webHostPort              web port on docker host
+     * @param debugPort                debug port on docker
+     * @param debugHostPort            debug port on docker host
+     * @param brokerInternalDebugPort
+     * @param dockerImage              full name of image (e.g. "kapua/kapua-broker:1.1.0-SNAPSHOT")
+     * @return Container configuration for specific boroker instance
+     */
+    private ContainerConfig getBrokerContainerConfig(String brokerAddr, String brokerIp,
+            String clusterName,
+            String controlMessageForwarding,
+            int mqttPort, int mqttHostPort,
+            int mqttsPort, int mqttsHostPort,
+            int webPort, int webHostPort,
+            int debugPort, int debugHostPort,
+            int brokerInternalDebugPort,
+            String dockerImage) {
+
+        final Map<String, List<PortBinding>> portBindings = new HashMap<>();
+        addHostPort("0.0.0.0", portBindings, mqttPort, mqttHostPort);
+        addHostPort("0.0.0.0", portBindings, mqttsPort, mqttsHostPort);
+        addHostPort("0.0.0.0", portBindings, webPort, webHostPort);
+        addHostPort("0.0.0.0", portBindings, debugPort, debugHostPort);
+
+        final HostConfig hostConfig = HostConfig.builder().portBindings(portBindings).build();
+
+        List<String> envVars = Lists.newArrayList("commons.db.schema.update=true",
```

Comment:
as a general annotation I'd like to have all these parameters configurable from outside.
If it's not a so easy change to do we could merge this pr as it is now and open a new issue for making this parameters configurable.
What do you think?